arch/arm/boot/dts/overlays/revpi-core-1-1-overlay.dts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Device tree overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Core 1.1
+ */
+
+#include "revpi-core-overlay.dts"
+#include <dt-bindings/interrupt-controller/irq.h>
+
+/{
+	compatible = "brcm,bcm2711", "brcm,bcm2837";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			compatible = "kunbus,revpi-core-1-1", "brcm,bcm2711";
```

Comment:
```suggestion
			compatible = "kunbus,revpi-core-2022", "brcm,bcm2711";
```

Reply:
compatible = "kunbus,revpi-core-2022", "brcm,bcm2711";

Reply:
No, this is Core 3 not Core S